### ISSUE TITLE
Multi-part comments in PGN

### ISSUE THREAD

**Issue by yo35** [yo35/kokopu#40]
When parsing a PGN with text comments split into several `{...}` parts such as:

```
1. e4 e6 {comment-A-part-1} (1... c6) {comment-A-part-2} 2. d4 d5 {comment-B-part-1} {comment-B-part-2} 3. e5 *
```

... the last part erases the previous ones during parsing.

Expected behavior: merge the parts related to the same move, so that the output game is equivalent to what would be obtained with:

```
1. e4 e6 {comment-A-part-1 comment-A-part-2} (1... c6) 2. d4 d5 {comment-B-part-1 comment-B-part-2} 3. e5 *
```

**Comment by yo35** [comment]
See also https://github.com/yo35/rpb-chessboard/issues/260

**Comment by yo35** [comment]
Fixed in 4.6.1
